parsing/HACKING.adoc

```diff
@@ -0,0 +1,9 @@
+link:parsing/parsetree.mli[Parsetree] and link:parsing/asttypes.mli[Asttypes]::
+Parsetree is an AST of the surface language of OCaml. It is well
```

Comment:
"surface language of OCaml"? Is there any reason not to sau "AST of the OCaml language"?